knowage-vue/src/modules/documentExecution/dashboard/DashboardController.vue

```diff
@@ -1,7 +1,7 @@
 <template>
     <div v-if="model" class="dashboard-container" :id="`dashboard_${model.configuration.id}`">
         <Button icon="fas fa-square-check" class="p-m-3 p-button-rounded p-button-text p-button-plain" style="position: fixed; right: 0; z-index: 999; background-color: white; box-shadow: 0px 2px 3px #ccc" @click="selectionsDialogVisible = true" />
-
+        <Button icon="fas fa-terminal" class="p-m-3 p-button-rounded p-button-text p-button-plain" style="position: fixed; right: 40px; z-index: 999; background-color: white; box-shadow: 0px 2px 3px #ccc" @click="logStuff" />
```

Comment:
Reminder: Remove logging elements from UI and code

Reply:
I will look into it.

Reply:
Removed logs DashboardController.vue (including Button for logging the dashboard).